cli_reference/developer_cli_odo/creating-applications-by-using-devfiles.adoc

```diff
@@ -0,0 +1,26 @@
+[id="creating-applications-by-using-devfiles"]
+= Creating applications by using devfiles
+include::modules/developer-cli-odo-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: creating-applications-by-using-devfiles
+toc::[]
+
+:FeatureName: Creating applications by using devfiles with `{odo-title}`
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+include::modules/developer-cli-odo-about-devfiles-in-odo.adoc[leveloffset=+1]
+
+== Creating a Java application by using a devfile
+
+.Prerequisites
+
+* `{odo-title}` is installed. 
+* You must know your ingress domain cluster name. Contact your cluster administrator if you do not know it. For example, `apps-crc.testing` is the cluster domain name for https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[Red Hat CodeReady Containers]
+* Experimental mode is enabled. To enable experimental mode in odo preferences, run:
```

Comment:
s/experimental mode/Experimental mode/ ? not sure

Reply:
I've checked how the team and users refer to it in upstream - indeed, it is capitalized. Fixed it.